src/NosCore.Controllers/CommandPacketController.cs

```diff
@@ -363,14 +362,14 @@ public void CreateItem(CreateItemPacket createItemPacket)
                 return;
             }
 
-            if (iteminfo.IsColored || iteminfo.Effect == boxEffect)
+            if (iteminfo.IsColored || iteminfo.Effect == Shared.Enumerations.Items.ItemEffectType.BoxEffect)
             {
                 if (createItemPacket.DesignOrAmount.HasValue)
                 {
                     design = (byte)createItemPacket.DesignOrAmount.Value;
                 }
 
-                rare = createItemPacket.Upgrade.HasValue && iteminfo.Effect == boxEffect
+                rare = createItemPacket.Upgrade.HasValue && iteminfo.Effect == Shared.Enumerations.Items.ItemEffectType.BoxEffect
```

Comment:
"Shared.Enumerations.Items" should not be necessary

Reply:
Okay